server/lib/category/delete-category/controller.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayEvent, APIGatewayProxyCallback, Context } from "aws-lambda";
+import { handleError, handleResult } from "shared";
+import { DeleteCategoryService } from "./delete-category-service";
+import { DeleteCategoryRequest } from "../model";
+
+
+export const handler = async (event: APIGatewayEvent, _: Context, callback: APIGatewayProxyCallback) => {
+    try {
+        const categoryId = event.pathParameters?.categoryId;
+        const email = event.queryStringParameters?.email;
+
+        if (!categoryId || !email) {
```

Comment:
Pointless to validate this , because in service you already validate using joi, remove.

Reply:
Why resolved , you needed to delete it.